Gemfile

```diff
@@ -17,6 +17,7 @@ end
 
 group :system_tests do
   gem 'voxpupuli-acceptance', '~> 1.0',  :require => false
+  gem 'pry',                             :require => false
```

Comment:
please remove this, the Gemfile is autogenerated and we want to keep it identical across our modules.

Reply:
I've added this as now the acceptance tests are failing with the following error:

```
 An error occurred while loading ./spec/acceptance/cs_primitive_spec.rb. - Did you mean?
                    rspec ./spec/acceptance/cs_commit_spec.rb
                    rspec ./spec/acceptance/cs_order_spec.rb
                    rspec ./spec/acceptance/cs_group_spec.rb

Failure/Error: require 'pry'
LoadError:
  cannot load such file -- pry
  Did you mean?  pty
  ```
  
  So this looks like something that's broken on the generator part?